pymatgen/io/adf.py

```diff
@@ -85,21 +85,15 @@ def __init__(self, name, options=None, subkeys=None):
         """
         Initialization method.
 
-        Parameters
-        ----------
-        name : str
-            The name of this key.
```

Comment:
Still noticed some non-Google styled docstrings. There might be some hidden format I'm not aware of. (Thought I searched keywords `Parameters` for some reason.)

Reply:
yeah, i noticed them too. `Args:` would be better. likewise

https://github.com/materialsproject/pymatgen/blob/ca8f608dec81743d506f365e91efd3bf76d468cf/pymatgen/io/adf.py#L186-L190

```diff
  Returns:
- -------
```

Reply:
I tried to fix as much as I could find in e0c1366553c6a9642bb5f0e0addccbec85745705 and 48a8fd53858f6dec1033c4f80e8b2347e560789b 😄

Reply:
looks good!

Reply:
Thanks. Let me know if I'm missing any.